device_services/generic/numeric_sensor.md

```diff
@@ -84,7 +86,8 @@ The most popular sensors measure temperature, air humidity, or light intensity.
     "sub_value": "C"
```

Comment:
I dont know if it'll work with the subvalue "c", because every index would be replaced with a new one?